soc/xtensa/intel_adsp/common/boot.c

```diff
@@ -45,7 +45,11 @@
 #elif defined(CONFIG_SOC_SERIES_INTEL_CAVS_V20)
 #define PLATFORM_INIT_HPSRAM
 #define PLATFORM_INIT_LPSRAM
+#if defined(CONFIG_BOARD_INTEL_ADSP_CAVS20_JSL)
+#define PLATFORM_HPSRAM_EBB_COUNT 16
```

Comment:
This is another situation where we've got some legacy hardware parameters still sitting in C code.  Ideally we want this value to be an attribute on a devicetree node associated with hpsram (which we have, but all it stores right now is the byte size).

Reply:
@andyross I was thinking the same. I can take a look at replacing this with a DT look-up (in a separate patch).